documentation/practices.md

```diff
@@ -80,5 +80,43 @@ Configuration properties for content URLs are:
+
+This component requires the following parameters:
+* `render`: The list of the elements to display from Indi. For example, `render={['carousel', 'brand', 'product']}`.
+Note:  Provide the correct configurations to render the element  correctly.
+* `configuration`: The structured object that consists of the configuration required for the Indi components. These configurations are defined in the `indi` parameter in `react-pwa-reference-storefront⁩/ep.config.json' file.
+* `keywords`: A string of the keywords for the product in which the Indi component is displayed.
+
+Static strings for the Indi component are localized in the Reference Storefront and are overridden whenever the Indi component is used. With this setting, the component displays the storefront in the locale that a customer chooses. The localized strings for the Indi component can be found in the corresponding language files within `src/localization`, for strings with the prefix `indi-`.
+
+The following are the configuration settings for the indi parameters in the `ep.config.json` file:
+
+| **Functionality**| **Parameter**| **Description**|
+|--|--|--|
+| **Integration**|`indi.enable`| The parameter that enables the component for the integration with Indi. For more information about the required configurations, see the company [website](https://indi.com/).|
+| **Theming**|`indi.carousel`| The configurations for the Indi carousel component.|
+| |`indi.carousel.apikey`| The _apikey_ that connects a carousel to Indi.|
+| |`indi.carousel.id`| The id used to connect the carousel to Indi.|
+| |`indi.carousel.size`| The parameter used to theme the carousel that displays the content from Indi.|
+| |`indi.carousel.theme`| The parameter used to theme the carousel that displays the content from Indi.|
+| |`indi.carousel.round_corners`| The parameter used to theme the carousel that displays the content from Indi.|
+| |`indi.carousel.show_title`| The parameter used to theme the carousel that displays the content from Indi.|
+| |`indi.carousel.show_views`| The parameter used to theme the carousel that displays the content from Indi.|
+| |`indi.carousel.show_likes`| The parameter used to theme the carousel that displays the content from Indi.|
+| |`indi.carousel.show_buzz`| The parameter used to theme the carousel that displays the content from Indi.|
+| |`indi.carousel.animate`| The parameter used to theme the carousel that displays the content from Indi.|
+| **Product Review**|`indi.productReview`| Configurations for the Indi product review component.|
+| |`indi.productReview.submit_button_url`| The URL provided by Indi which directs to the action performed when the submit button is clicked on a product review.|
+| |`indi.productReview.thumbnail_url`| The thumbnail provided by Indi for the submit button.|
+| **Brand Ambassador**|`indi.brandAmbassador`| Configurations for the Indi brand ambassador component.|
```

Comment:
```suggestion
| **Brand Ambassador**|`indi.brandAmbassador`| The configurations for the Indi brand ambassador component.|
```

Reply:
Is the component name brand ambassador?

Reply:
The config for it is

Reply:
configuration for brand ambassador component in Indi, right?

Reply:
yes, that's correct